sentry-ruby/sentry-ruby.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
 
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "concurrent-ruby", '~> 1.0', '>= 1.0.2'
+  spec.add_dependency "concurrent-ruby", "~> 1.0", ">= 1.0.2"
+  spec.add_dependency "mime-types", "~> 3.0"
```

Comment:
ugh sad that this isn't in stdlib, we are actually very strict about introducing deps. 
Since we are sending the `filename` anyway, relay could in theory do the MIME extraction server side, so I would leave this out of the client and remove the `infer_content_type` above. It's only a nice to have and I would rather not add the new dependency.